test/functional/feature_addrman.py

```diff
@@ -68,17 +68,16 @@ def run_test(self):
             self.start_node(0, extra_args=["-checkaddrman=1"])
         assert_equal(self.nodes[0].getnodeaddresses(), [])
 
-        self.log.info("Check that addrman from future cannot be read")
+        self.log.info("Check that addrman from future is overwritten with new addrman")
         self.stop_node(0)
         write_addrman(peers_dat, lowest_compatible=111)
```

Comment:
Is it possible to do this using some hidden RPC?

Reply:
Nope. This is a python function that literally just writes a dummy (but valid) peers.dat file to disk.